tools/ctl/ctl.go

```diff
@@ -247,12 +247,17 @@ var getResults = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		fireClient, err := rundex.NewFirestore(cmd.Context(), *project)
-		if err != nil {
-			log.Fatal(err)
+		var dex rundex.Reader
```

Comment:
on second look, this is a change to getResults which is more about viewing results locally, not running local benchmarks. so this, too, should be split into a separate change, right?